scripts/dockerdev.sh

```diff
@@ -43,7 +43,7 @@ dockerdev () {
            --add-host="dev1.shiftcrypto.ch:176.9.28.155" \
            --add-host="dev2.shiftcrypto.ch:176.9.28.156" \
            -v $repo_path:/opt/go/src/github.com/digitalbitbox/bitbox-wallet-app \
-           bitbox-wallet bash
+           shiftcrypto/bitbox-wallet-app bash
```

Comment:
There's also `echo "No bitbox-wallet docker image found! ..."` message above. Maybe adjust that as well? It greps for `bitbox-wallet`. I would change that too, to `bitbox-wallet-app`.

Reply:
:heavy_check_mark: done, good catch